### PR TITLE
Create DEVELOPMENT.md

### PR DESCRIPTION
**Pull request by OisinNolan**

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Tests are run using [Jasmine](https://jasmine.github.io/) and [Karma](https://karma-runner.github.io/latest/index.html).
+- To run tests, enter:
+```
+$ npm run test
+```
+- Any new features added to AccName should be appropriately tested with unit tests in a corresponding `_test.ts` file in the same directory as the edited file. For example, tests for `lib/ruleA.ts` should be in `lib/ruleA_test.ts`.
+### Code Formatting
+- [Google TypeScript Style](https://www.npmjs.com/package/gts) (GTS) is used to format the TypeScript code for AccName. The code is automatically formatted using a commit-hook. This means that the code is often automatically changed by GTS when it is committed, in which case you can simply `$ git add .` and `$ git commit --amend` to update your commit with the newly-formatted code.
+- AccName uses [JSDoc](https://jsdoc.app/) style comments.
+### Performance
```
Comment (AlexLloyd0) [inline DEVELOPMENT.md:24]: When I read performance I think speed/efficiency. Maybe 'regressions' or something is better? IDK it's also not great...

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+It is important that contributions to AccName do not have a negative impact on its performance. Performance is currently being measured using the [AccName Comparison App](https://github.com/googleinterns/accessible-name/tree/master/validation). Specifically, we use the following metrics:
+- % of [Web Platform Tests](http://wpt.live/accname/) passed.
+- % Nodes in real world web-pages for which AccName is likely incorrect. This is defined as the % of Nodes for which AccName produces an accessible name of X with all comparison implementations producing an accessible name of Y. In the context of the Comparison App, this can be described with an agreement grouping of the following form: `{{chrome, aom, axe, bg},{accname}}`.
```
Comment (AlexLloyd0) [inline DEVELOPMENT.md:27]: This section doesn't tell me how I know if my PR is safe. What if it causes a regression in WPT but improvement in field tests? What if the spec changes but no other implementations have updated yet? We'll appear to be performing worse on field tests.

You don't need to answer all those questions. But it should be clear for contributors what to do. Maybe something like "Include the % of WPT passed and the results of a field test on <urls> both before and after your PR"

File: DEVELOPMENT.md (hunk 1 above)
Comment (OisinNolan) [reply DEVELOPMENT.md:24]: How about 'Accuracy', 'Output Accuracy' or similar ?

Comment (AlexLloyd0) [reply DEVELOPMENT.md:24]: 'Ouptut Accuracy' works :)

File: DEVELOPMENT.md (hunk 2 above)
Comment (AlexLloyd0) [reply DEVELOPMENT.md:27]: As we don't have CI for this yet, could you address this comment in the PR? :)

**Review by AlexLloyd0 — APPROVED** [review]

**Pull request merged**